core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -106,6 +110,36 @@ class VehiclesResource(db: Database, rviClient: RviClient)
         rviClient.sendMessage(s"genivi.org/vin/${vin}/sota/getpackages", io.circe.Json.Empty, ttl())
         // TODO: Confirm getpackages in progress to vehicle?
         complete(NoContent)
+      } ~
+      pathPrefix("updates") {
```

Comment:
We'll have to move this to a 'Vehicle API' service. Let's put a comment here and start a new task.